app/models/student.rb

```diff
@@ -9,7 +9,7 @@ class Student < ApplicationRecord
   has_many :assessments
 
   has_many :enrollments
-  has_many :coordinators, through: :enrollments
```

Comment:
A student can have a coordinator through an affiliate. But imagine that a coordinator is fired... Do you think it makes sense to go update 200 students to the new coordinator? No, that's silly.